uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/leave-type-wizard/components/leave-type-wizard.component.js

```diff
@@ -138,6 +139,34 @@ define([
       watchFieldAllowCarryForward();
     }
 
+    function initValidatorForField (field) {
+      $scope.$watch(function () {
+        return field.value;
+      }, function (value) {
+        delete field.error;
+
+        if (value === undefined || value === '') {
+          return;
+        }
+
+        field.validations.forEach(function (validation) {
+          if (!validation.rule.test(value)) {
+            field.error = validation.message;
+          }
+        });
+      });
+    }
+
+    function initValidators () {
```

Comment:
Doc missing.

Reply:
Yes, sorry, you may looked through commits, I added a documentation later in commits.